core/services/cable_guy/main.py

```diff
@@ -64,15 +63,7 @@ def configure_interface(interface: EthernetInterface = Body(...)) -> Any:
     version="1.0.0",
     prefix_format="/v{major}.{minor}",
 )
-app.mount("/static", StaticFiles(directory=str(STATIC_FOLDER)), name="static")
-
-
-@app.get("/", response_class=HTMLResponse, summary="Cable Guy web interface")
-def index() -> Any:
-    """REST API endpoint to retrieve web application for Cable Guy."""
-    index_file_path = Path.joinpath(HTML_FOLDER, "index.html")
-    return open(index_file_path, "r").read()
-
+app.mount("/", StaticFiles(directory=str(STATIC_FOLDER), html=True))
```

Comment:
It should be pointing to the HTML_FOLDER here, not the STATIC_FOLDER, right?

Reply:
Done.